packages/components/src/Markdown/Markdown.tsx

```diff
@@ -9,7 +9,7 @@ interface MarkdownProps {
   /**
    * Rich text content.
    */
-  readonly content: string;
+  readonly content?: string;
```

Comment:
Making this prop optional would allow `Markdown` to be used without any content. That shouldn't ever be a use case. for the markdown component.

Reply:
How would you handle an optional message prop in the Confirmation Modal with a required content prop?

Reply:
Added!